port/common/omrport.tdf

```diff
@@ -1114,6 +1114,9 @@ TraceExit=Trc_PRT_sysinfo_processor_disable_feature_Exit Group=sysinfo Overhead=
 TraceEntry=Trc_PRT_sysinfo_processor_set_feature_Entered Group=sysinfo Overhead=1 Level=5 NoEnv Template="sysinfo_processor_set_feature: desc = %p, feature = %d, enable = %d"
 TraceExit=Trc_PRT_sysinfo_processor_set_feature_Exit Group=sysinfo Overhead=1 Level=5 NoEnv Template="sysinfo_processor_set_feature: returning with %zd"
 
+TraceEntry=Trc_PRT_sysinfo_get_processor_feature_name_Entered Group=sysinfo Overhead=1 Level=5 NoEnv Template="sysinfo_get_processor_feature_name: feature = %d"
+TraceExit=Trc_PRT_sysinfo_get_processor_feature_name_Exit Group=sysinfo Overhead=1 Level=5 NoEnv Template="sysinfo_get_processor_feature_name: returning with %s"
+
```

Comment:
New tracepoints should be added to the end of the file so existing tracepoint numbers remain stable.
See https://github.com/eclipse/openj9/blob/master/doc/diagnostics/AddingTracepoints.md.

Reply:
Is it too late now to open a new PR to change this?

Reply:
made a new PR here: https://github.com/eclipse/omr/pull/5422